config/secrets.yml

```diff
@@ -16,6 +16,16 @@ default: &default
   aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
   aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
   aws_bucket: <%= ENV['AWS_BUCKET'] %>
+  enable_persona: <%= ENV['ENABLE_PERSONA'] || true %>
```

Comment:
well put. I agree. I've been wanting to have a CONFIGURATION.md  file which
explains all the ENV settings one can use to configure the app. (and
supporting that.. ensuring that all configuration is done via ENV)

On Wed, 20 May 2015 at 13:25 James Kiesel <notifications@github.com> wrote:

> In config/secrets.yml
> <https://github.com/loomio/loomio/pull/2172#discussion_r30664769>:
>
> > @@ -16,6 +16,16 @@ default: &default
> >    aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
> >    aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
> >    aws_bucket: <%= ENV['AWS_BUCKET'] %>
> > +  enable_persona: <%= ENV['ENABLE_PERSONA'] || true %>
>
> <anything> || true, is always going to be true, so we can't default it to
> true like that. (And <anything> || false will just give the value of
> anything if it exists, and a falsey value otherwise, so that' not that
> useful either.)
>
> For defaults, any logic in yml files is a little smelly to me; maybe
> there's a separate helper / settings file which wraps
> Rails.application.secrets? I just remember when we tried to add defaults
> a while back, we kinda found we didn't need any.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2172/files#r30664769>.
>